common/changes/office-ui-fabric-react/v-vibr-PersonaFix2_2018-12-14-02-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona: fixing how coinProps are passed through from Persona to PersonaCoin.",
+      "type": "minor"
```

Comment:
Shouldnt this be a patch version btw?

Reply:
Not sure... the surface API file was modified so I would believe it's a minor. @cliffkoh can you comment on this before merge? Thanks!

Reply:
Minor seems right. The change is in the API file is backwards compatible but not forwards compatible.